src/createServer.js

```diff
@@ -1,3 +1,39 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('../src/convertToCase/convertToCase');
+const { validation } = require('./validation');
+const createServer = () => {
+  const server = http.createServer((req, res) => {
```

Comment:
Return 404 status code if the path does not match the expected format. This can be done by adding a conditional to check the pathname before proceeding.